File: JobSearchingWebApp/Endpoints/Kandidat/GetById/KandidatGetByIdResponse.cs
using JobSearchingWebApp.Database;

namespace JobSearchingWebApp.Endpoints.Kandidat.GetById
{
    public class KandidatGetByIdResponse : Korisnik
    {
        public string Ime { get; set; }

        public string Prezime { get; set; }

        public DateTime DatumRodjenja { get; set; }

        public string MjestoPrebivalista { get; set; }

        public string Zvanje { get; set; }
    }
}

File: front-end/src/app/endpoints/kandidat-endpoint/get-by-id/kandidat-get-by-id-response.ts
export interface KandidatGetByIdResponse {
  ime: string,
  prezime: string,
  datumRodjenja: Date,
  mjestoPrebivalista: string,
  zvanje: string,
  userName: string,
  email: string,
  phoneNumber: string,
  phoneNumberConfirmed: boolean,
  twoFactorEnabled: boolean,
  emailConfirmed: boolean
}
